src/utils/convert-filter.ts

```diff
@@ -15,8 +15,13 @@ export const convertFilter = (filter) => {
     const { property, value } = filterProperty
     switch (property.type()) {
     case 'string':
+      let regexPattern = '';
+      if (property.name() === 'status')
```

Comment:
This won't do, the only way I see it works is if you somehow check whether the property is enum/has `availableValues`